client/blocks/domain-to-plan-nudge/index.jsx

```diff
@@ -80,9 +79,22 @@ class DomainToPlanNudge extends Component {
 		} );
 
 		page( `/checkout/${ siteId }/personal` );
+	};
+
+	renderRegular() {
+		const { siteId, translate } = this.props;
+		return (
+			<UpgradeNudge
+				title={ translate( 'Upgrade to a Personal Plan and Save!' ) }
+				message={ translate( 'Get live chat support and no ads bundled with your domain!' ) }
```

Comment:
I'm mostly trying to keep the text to a single line, let's see if folks have some suggestions.